src/test/scala/chiseltest/coverage/Test1Module.scala

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
It doesn't appear this is being used other than in SimulatorCoverageTest, can it be put in that same file?

Reply:
I would rather not, since once I add the line coverage, I will depend on exact line number in this file for my tests.
So yes, in isolation it would make sense to move this into its own file, but considering the other additions I want to make, it wouldn't be a good idea.

The one thing I can offer is to make a separate package, something along the lines of `chiseltest.coverage.circuits` or `chiseltest.coverage.examples` or `chiseltest.coverage.benchmarks` or similar.

Reply:
I see. The main benefit of having it in the same file would be marginally more convenient readability. But a comment describing the line stability concerns would be great.

Reply:
added a comment